include/vki/vki-scnums-riscv64-linux.h

```diff
@@ -34,133 +34,133 @@
 // is a #include of
 //      linux-5.10.4/include/uapi/asm-generic/unistd.h
 
-#define __NR_getxattr 8
```

Comment:
VKI (Valgrind Kernel Interface) files contain mostly copies of definitions from include files of a given target operating system. Upstream Valgrind keeps them generally formatted in the same way as they are present in the original files. I think it is done to simplify their future comparison. While I realize it is not really that important for these syscall numbers, it still looks sensible to me to follow that convention and be consistent with the upstream.

Reply:
Thanks for the review, fixed.